### ISSUE TITLE
Is it possible to enable login without passing credentials via stdio when server is started?

### ISSUE THREAD

**Issue by devforfu** [facebookresearch/visdom#659]
**Is your feature request related to a problem? Please describe.**
The `visdom` library has flat `-enable_login`. However, as far as I know, the only way to provide the credentials is via `stdin`. (See the [following fragment](https://github.com/facebookresearch/visdom/blob/1cb8c19cb9fb86b3930ab06ce6d19837fba81244/py/visdom/server.py#L1856-L1857)).

Would be great to have a possibility to load the credentials without manual interaction with CLI. It would be especially helpful example, when `visdom` server is launched from Jupyter Notebook like `!visdom -enable_login` without leaving browser.

**Describe the solution you'd like**
A possible solution could be to support a set of environment variables to simplify the process of credentials setup. For example, it could be implemented as the following snippet shows.
```python
if FLAGS.enable_login:
    use_env = os.environ.get('VISDOM_USE_ENV_CREDENTIALS', False)
    if use_env:
        username = os.environ.get('VISDOM_USERNAME')
        password = os.environ.get('VISDOM_PASSWORD')
        if not (username and password):
            ...  # raise an exception, fallback to stdio, etc.
    else:
        username = input("Please input your username: ")
        password = getpass.getpass(prompt="Please input your password: ")

    user_credential = {
        "username": username,
        "password": hash_password(hash_password(password))
    }

    if not os.path.isfile(DEFAULT_ENV_PATH + "COOKIE_SECRET"):
        set_cookie()
    elif FLAGS.force_new_cookie:
        set_cookie()
else:
    user_credential = None
```

**Describe alternatives you've considered**
Another alternative would be to support additional CLI flags that would allow to specify the `username` and `password` like `visdom -enable_login -username user -password secret`. Though probably this scenario is less safe than going with env vars.

**Comment by JackUrb** [comment]
Hi @devforfu - Seems like a reasonable solution to me. Feel free to open a PR and I can pull it in!

**Comment by devforfu** [comment]
@JackUrb Done! Please let me know if it looks good or something should be done in a different way.

**Comment by devforfu** [comment]
Solved with #660.
